### DISCUSSION TITLE
Semester 4 Week 1

### DISCUSSION THREAD

**Discussion by Challenger3010** [SE-TINF22B6/CookHub#123] [Show and tell]
### Backend
✅ We improved our database schema in the backend
    1. It is now possible to provide a specific unit of measure for each ingredient quantity of a recipe
    2. There are determaned 24 diffrent categories and it is possible for one recipe to be in multiple categories
    3. Some bug fixes
    4. It is possible to specify a difficulty for a recipe
    5. Recipe can have multiple adventure text created by OpenAI instead of just one
✅ Our very competent backend team accomplished the impossible by creating endpoints for recipes. 

### Frontend
✅ It is possible to send requests to the backend enpoints for recipes 
1. We implemented a search function for recipes you are now able to search all recipes in the database by name and get a specific Recipe by ID or search for all.
2. Major Bug Fixes
3. Rage quit button that sends you straight to Lieferando

### The Future
🚧 Adventure text created by OpenAI 🚧 
🚧 Fix UI issues 🚧 
🚧 Fill Recipe Page with backend data🚧 

See you next time! <img src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/ffa3f6d2-0d92-441e-9d78-51a62202b6e2" alt="carlos" align="right" width="100" />

**Your**   <img width="50" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/1a218c37-d3b0-4c88-9ae1-827b5a4eba42">   **Team**

**Comment by fehlend0** [comment]
Wow, it sounds like your team had an incredibly productive week! Improving the database schema with new features like specific units of measure and multiple categories for recipes shows great attention to detail. And kudos to the backend team for creating those essential endpoints. The frontend enhancements, especially the search function and bug fixes, will surely enhance user experience. Exciting plans for the future too, with Adventure text by OpenAI and UI fixes on the horizon. Keep up the excellent work!
